easybuild/easyconfigs/p/PyWBGT/PyWBGT-1.0.0-foss-2021b.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'PythonBundle'
+
+name = 'PyWBGT'
+version = '1.0.0'
+
+homepage = 'https://github.com/QINQINKONG/PyWBGT'
+description = """Cython source code for estimating wet bulb globe temperature (WBGT) from datasets of standard
+meterological measurements using models developed by Liljegren et al (2008)"""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+toolchainopts = {'openmp': True}
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('SciPy-bundle', '2021.10'),
+    ('xarray', '0.20.1'),
+]
+
+use_pip = True
+
```

Comment:
this fixes my failure:
```suggestion
# remove .c file extension from text files, to avoid that GCC thinks they are C source code
local_common_preinstallopts = "mv WBGT.c WBGT && mv coszenith.c coszenith && "
```